src/main/java/io/anserini/ltr/DumpTweetsLtrData.java

```diff
@@ -125,7 +125,7 @@ private DumpTweetsLtrData() {}
 
       TopDocs rs = searcher.search(q, args.hits);
       List<String> queryTokens = AnalyzerUtils.tokenize(new TweetAnalyzer(), queryString);
-      RerankerContext context = new RerankerContext(searcher, query, queryString.toString(), queryString,
+      RerankerContext<String> context = new RerankerContext<>(searcher, query, queryString, queryString,
```

Comment:
Should be `<Integer>` since this is for MB topics of which the query ids are integers